src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/UseEnvironmentProcessIdTests.cs

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Xunit;
+using VerifyCS = Test.Utilities.CSharpCodeFixVerifier<
+    Microsoft.NetCore.Analyzers.Runtime.UseEnvironmentProcessId,
+    Microsoft.NetCore.Analyzers.Runtime.UseEnvironmentProcessIdFixer>;
+using VerifyVB = Test.Utilities.VisualBasicCodeFixVerifier<
+    Microsoft.NetCore.Analyzers.Tasks.DoNotCreateTaskCompletionSourceWithWrongArguments,
+    Microsoft.NetCore.Analyzers.Tasks.DoNotCreateTaskCompletionSourceWithWrongArgumentsFixer>;
```

Comment:
This needs to be updated. And probably it's worth adding tests for vb.

Reply:
Updated to what?

Reply:
To the analyzer and code fix under test, which is use env process id.
@stephentoub

Reply:
Ah, yes :)